docs/design/specs/Ecma-335-Augments.md

```diff
@@ -885,19 +886,25 @@ Add a third bullet
     ...
   }
 }
+.class D extends C
+{
+  .method virtual DerivedRetTypeNotDerivedFromMoreDerivedRetType VirtualFunction() {
+    .custom instance void [System.Runtime]System.Runtime.CompilerServices.PreserveBaseOverridesAttribute::.ctor() = ( 01 00 00 00 ) 
+    .override A.VirtualFuncion 
+    ...
+  }
+}
 ```
 For this example, the behavior of calls on objects of various types is presented in the following table:
-| Type of object | Method Invocation | Method Called |
-| ---  | --- | --- |
-| A | A::VirtualFunction() | A::VirtualFunction |
-| B | A::VirtualFunction() | B::VirtualFunction |
-| B | B::VirtualFunction() | B::VirtualFunction |
-| C | A::VirtualFunction() | C::VirtualFunction |
-| C | B::VirtualFunction() | C::VirtualFunction |
-| C | C::VirtualFunction() | C::VirtualFunction |
+| Type of object | Method Invocation | Method Called | Notes |
+| ---  | --- | --- | --- |
+| A | A::VirtualFunction() | A::VirtualFunction | |
+| B | A::VirtualFunction() | B::VirtualFunction | |
+| B | B::VirtualFunction() | B::VirtualFunction | |
+| C | A::VirtualFunction() | C::VirtualFunction | |
+| C | B::VirtualFunction() | C::VirtualFunction | |
+| C | C::VirtualFunction() | C::VirtualFunction | |
+| D | B::VirtualFunction() | ERROR | A program containing type D is not valid, as B::VirtualFunction would be implemented by D::VirtualFunction which is not *covariant-return-compatible-with* (§I.8.7.1) B::VirtualFunction |
 "
-
```

Comment:
Wow, I didn't expect such a profound effect on the wording, I really appreciate your diligence towards polishing the spec!